pkgs/games/openttd/default.nix

```diff
@@ -18,6 +20,12 @@ let
     sha256 = "0qnmfzz0v8vxrrvxnm7szphrlrlvhkwn3y92b4iy0b4b6yam0yd4";
   };
 
+  playmidi = writeScriptBin "playmidi" ''
+    #!/bin/sh
+    trap "${procps}/bin/pkill fluidsynth" EXIT
+    ${fluidsynth}/bin/fluidsynth -a ${audioDriver} -i ${soundfont-fluid}/share/soundfonts/FluidR3_GM2-2.sf2 $*
```

Comment:
What will `fluidsynth` choose if no audio driver is specified? Is there some kind of "auto" option?